openapi-validator/src/main/java/io/ballerina/openapi/validator/ServiceValidator.java

```diff
@@ -376,24 +377,33 @@ private DiagnosticSeverity extractOpenAPIAnnotation(DiagnosticSeverity kind, Fil
                                 fieldNode.location());
                         validations.add(diagnostic);
                     }
-                } else if (specificFieldNode.fieldName().toString().trim().equals("failOnErrors")) {
+                } else if (specificFieldNode.fieldName().toString().trim().equals(Constants.FAILONERRORS)) {
                     String failOnErrors = openAPIAnnotation.toString();
-                    if (failOnErrors.trim().equals("false")) {
+                    if (failOnErrors.trim().equals(FALSE)) {
                         kind = DiagnosticSeverity.WARNING;
                         filters.setKind(kind);
                     }
                 } else if (openAPIAnnotation instanceof ListConstructorExpressionNode) {
                     ListConstructorExpressionNode list = (ListConstructorExpressionNode) openAPIAnnotation;
-                    String attributeName = specificFieldNode.fieldName().toString().trim();
+                    Node field = specificFieldNode.fieldName();
+                    String attributeName = ((Token) field).text();
                     switch (attributeName) {
-                        case "tags":
+                        case Constants.TAGS:
                             List<String> tags = setFilters(list);
                             filters.setTag(tags);
                             break;
-                        case "excludeTags":
+                        case Constants.EXCLUDETAGS:
                             List<String> eTags = setFilters(list);
                             filters.setExcludeTag(eTags);
                             break;
+                        case Constants.OPERATIONS:
+                            List<String> operations = setFilters(list);
+                            filters.setOperation(operations);
+                            break;
+                        case Constants.EXCLUDEOPERATIONS:
```

Comment:
Please use underscore to separate words when defining constants (i.e. EXCLUDE_OPERATIONS). Check all the other constant definitions. 

Also please try to use context aware names (using a prefix/postfix) as much as possible, when defining constants (i.e. `ATTRIBUTE_TAGS`, `ATTRIBUTE_EXCLUDE_TAGS`) 🙂